### ISSUE TITLE
bug: dashboard keybind for `project` extra

### ISSUE THREAD

**Issue by andresatierf** [LazyVim/LazyVim#1810]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

v0.9.4

### Operating system/version

Linux 5.15.133-1-MANJARO

### Describe the bug

the `Projects` keybind still has the brackets
![2023-10-19_21-01-17_grim](https://github.com/LazyVim/LazyVim/assets/29847057/34964fab-ffce-481f-bbbd-8fc84b01f9a1)

I tried to change the extra here
https://github.com/LazyVim/LazyVim/blob/0bb628053beec25efbc5bd1e8600ef039fa75a20/lua/lazyvim/plugins/extras/util/project.lua#L49-L61
to this
```lua
local projects = {
  action = "Telescope projects",
  desc = " Projects",
  icon = " ",
  key = "p",
  key_format = "  %s",
}
```
to open a pr but it didn't fix the issue

### Steps To Reproduce

- enable `util.project` extra
- open dashboard

### Expected Behavior

all the keys to be formatted the same

### Repro

```Lua
-- DO NOT change the paths and don't remove the colorscheme
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", lazypath, })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  "folke/tokyonight.nvim",
  "folke/LazyVim",
  -- add any other plugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

vim.cmd.colorscheme("tokyonight")
-- add anything else here
```

**Comment by dpetka2001** [comment]
Will be solved by #1809.

**Comment by folke** [comment]
Fixed
